sbc_compassion/models/import_letter_line.py

```diff
@@ -92,3 +109,23 @@ def _set_name(self):
                 line.name = str(
                     line.sponsorship_id.partner_codega) + " - " + str(
                         line.child_code)
+
+    @api.multi
+    def save_lines(self, mandatory_review=False):
+        """
+        """
+        ids = []
+        for letter in self:
+            dict_ = {
```

Comment:
Since `dict` is a reserved name in python, I think `vals` would be less similar.